theme/theme.go

```diff
@@ -136,7 +136,7 @@ func (t *builtinTheme) TextSize() int {
 }
 
 func loadCustomFont(env, variant string, fallback fyne.Resource) fyne.Resource {
-	variantPath := strings.Replace(env, "Regular", variant, 0)
+	variantPath := strings.Replace(env, "Regular", variant, 0) // n == 0 gives no results. Wrong value?
```

Comment:
No the reviewer:
- Having the n == 0 in the function results in no result. Is this set up correctly?

Reply:
It should have been -1